module/settings.js

```diff
@@ -234,4 +234,28 @@ export const registerSystemSettings = async function () {
     default: true,
     config: true
   })
+
+  /**
+   * Automatically add warrior's level to the initiative value
+   */
+    game.settings.register('dcc', 'automateWarriorInititative', {
```

Comment:
Please can you correct the spelling of 'Inititative' in the setting (the localisation key is correct, but the setting is not).

Reply:
Fixed